json_schemas/services-g-cloud-7-scs.json

```diff
@@ -81,14 +81,12 @@
       }
     },
     "priceMin":{
-      "type":"number",
-      "minimum":0
+      "type":"string",
+      "pattern": "^\\d+(?:\\.\\d{2})?$"
     },
     "priceMax":{
-      "type":[
-        "number",
-        "null"
-      ]
+      "type": "string",
+      "pattern": "^\\d+(?:\\.\\d{2})?$"
```

Comment:
I think prices were allowed up to four decimal places for G6.

Are we sure having numbers for G6 but strings for G7 will not cause any problems?